lib/js/components/Cards/CardExpandable/index.ts

```diff
@@ -0,0 +1 @@
+export * from './CardExpandable.spec';
```

Comment:
do we need to export tests?

Reply:
To check by me.

Reply:
I have no explanation for this, I have found no reason for this to stay in the project. In the corresponding `Card` folder there is no such thing. So It's going out.